### DISCUSSION TITLE
Pomysły na dalszy rozwój aplikacji

### DISCUSSION THREAD

**Discussion by MatPerowicz** [KlubJagiellonski/pola-ios#223] [Ideas]
Najczęściej proponowane przez użytkoników zmiany zapisujemy [na tej liście](https://github.com/KlubJagiellonski/pola-backend/issues?q=is%3Aissue+is%3Aopen+label%3Akind%3Aidea). Są to m.in. lista ulubionych firm, czy mapa polskich sklepów.

**Comment by czater** [comment]
https://github.com/KlubJagiellonski/pola-backend/issues/186 to można by zaimplementować lokalnie, nie trzeba backend do tego. Mogę się tym zająć o ile pojawiłby się jakiś design.

**Comment by MatPerowicz** [comment]
Ale którym pomysłem dokładnie? Bo jest ich kilka?

śr., 2 cze 2021, 09:18 użytkownik czater ***@***.***> napisał:

**Comment by czater** [comment]
`Możliwość zapisywania/gromadzenia oceny danej firmy, tak aby nie trzeba było jej za każdym razem skanować.`

**Comment by czater** [comment]
Rozumiem, że byłoby to coś al'a lista z nazwami, logami firm i ich oceną, którą można by scrollować etc? 🤔

**Comment by MatPerowicz** [comment]
Ok, dzięki. Sprawdzę jak Android Team stoi z czasem i dam znać.

śr., 2 cze 2021, 09:27 użytkownik czater ***@***.***> napisał:

**Comment by MatPerowicz** [comment]
Tak, coś takiego. Tylko pytanie czy to możliwe, ponieważ nie zbieramy
żadnych informacji o użytkownikach.

śr., 2 cze 2021, 09:28 użytkownik czater ***@***.***> napisał:

**Comment by czater** [comment]
Tak, do zapisywania danych lokalnie nie trzeba zbierać żadnych danych o użytkownikach.

**Comment by MatPerowicz** [comment]
Zajrzyj proszę na slacka. Właśnie wrzuciłem do tego prowizoryczny design.
Są też chętni żeby zrobić to na android. Na slacku to przegadamy

śr., 2 cze 2021 o 12:53 czater ***@***.***> napisał(a):

> Tak, do zapisywania danych lokalnie nie trzeba zbierać żadnych danych o
> użytkownikach.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/KlubJagiellonski/pola-ios/issues/219#issuecomment-852927079>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMOACOILO4OJ4KJB4OZ5VHLTQYERRANCNFSM455C5CLQ>
> .
>


-- 
Z wyrazami szacunku,
Mateusz Perowicz
